dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlanner.cs

```diff
@@ -94,7 +94,7 @@ public async Task<Plan> CreatePlanAsync(string goal, CancellationToken cancellat
         return plan;
     }
 
-    private SequentialPlannerConfig Config { get; }
+    private SequentialPlannerConfig _config { get; }
```

Comment:
nit: The property was renamed according to the field's rules. Its name follows camel case, and it is prefixed with `_`. So, if the intention of this change is to convert the property into a class field, please remove the remaining getter.

Reply:
fixed - thanks!